.github/workflows/build.yaml

```diff
@@ -1,7 +1,5 @@
 name: Build MyHoard
 
-permissions: read-all
-
```

Comment:
This reverts the hardening PR: https://github.com/aiven/myhoard/pull/75
I doubt that's a good thing.

Reply:
This change was originally done to for the junit publish, for which I had got an explicit "go" from security. But I think this is unnecessary now, so, good catch.

Also, note that #75 says:
```Set the default workflow permissions to read - because this workflow does not need to modify anything in the project.```

So, it's a correct security-by-default measure, but it was "hardening", not "fixing a security problem". If we need to relax permissions for good reasons, I see nothing preventing this.